lib/common/collections/wp_users/output.rb

```diff
@@ -28,18 +28,10 @@ def output(options = {})
     end
 
     def remove_junk_from_display_names
-      display_names = []
       self.each do |u|
-        display_name = u.display_name
-        unless display_name == 'empty'
-          display_names << display_name
-        end
-      end
-      junk = get_equal_string_end(display_names)
-      unless junk.nil? or junk.empty?
-        self.each do |u|
-          u.display_name ||= ''
-          u.display_name = u.display_name.sub(/#{Regexp.escape(junk)}$/, '')
+        unless u.display_name == 'empty'
```

Comment:
the methods you deleted were the "fallback" when the title could not be extracted. It took all usernames and remove the last chars which were the same. I think we should leave it in as a fallback since the chars you remove are totally theme dependent. Each theme can define another title format (even for each site)